x/emissions/keeper/inference_synthesis/nonce_mgmt.go

```diff
@@ -37,7 +37,8 @@ func SelectTopNReputerNonces(reputerRequestNonces *emissions.ReputerRequestNonce
 	// loop reputerRequestNonces
 	for _, nonce := range sortedSlice {
 		nonceCopy := nonce
-		if currentBlockHeight >= nonceCopy.ReputerNonce.BlockHeight+groundTruthLag {
+		// Only select when the ground truth is available.
+		if currentBlockHeight >= nonceCopy.ReputerNonce.BlockHeight+groundTruthLag+epochLength {
```

Comment:
what's the reasoning for the extra  epochLength? the ground truth lag should be enough, no?

Reply:
gt_lag is added extra to epochLength. Reasons are historical (before we had gt_lag, epochLength was the measure), but this can now be simplified, so only gt_lag is considered for reputation (gt_lag since nonce, regardless of epochLength); also adding a requirement that gt_lag is >= epochLength. 

This changes the current meaning so affects topic definitions, but looks cleaner.
Will apply.,